apps/obs-web/src/views/AttributeSearch-unused.vue

```diff
@@ -0,0 +1,173 @@
+<template>
```

Comment:
do we need this file? Is it being used at all? seems like maybe not.  Also it's causing a bunch of linting errors so probably good to get rid of it.